src/components/Trade/TradeInput.tsx

```diff
@@ -278,22 +293,34 @@ export const TradeInput = ({ history }: RouterProps) => {
                 }
               />
             </FormControl>
-            <Button
-              type='submit'
-              size='lg'
-              width='full'
-              colorScheme={error ? 'red' : 'blue'}
-              isLoading={isSubmitting || isSendMaxLoading || !!action}
-              isDisabled={!isDirty || !isValid || !!action || !wallet}
-              style={{
-                whiteSpace: 'normal',
-                wordWrap: 'break-word'
-              }}
+            <Tooltip
+              label={translate('common.insufficientFunds')}
+              fontSize='md'
+              px={4}
+              hasArrow
+              isDisabled={
+                !isValid || isSubmitting || isSendMaxLoading || !!action || hasValidBalance
+              }
```

Comment:
Non-blocking, but I think it would make sense to use red errored state submit button with insufficient funds copy vs. a tooltip, for consistency across the app.